src/xbot/common/properties/DoubleProperty.java

```diff
@@ -28,10 +34,12 @@ public void set(double value) {
     }
     
     /**
-     *
+     * We only save the property if it's from a persistent type
      */
     public void save() {
+    	if(persistenceType == PropertyPersistenceType.Persistent) {
         permanentStore.setDouble(key, randomAccessStore.getDouble(key).doubleValue());
```

Comment:
There is a real tab character there; should make sure to eradicate those. @riverole Did you set up the checkstyle and formatting policies in your eclipse installation yet?